doc/api/repl.md

```diff
@@ -321,17 +326,17 @@ The following example shows two new commands added to the REPL instance:
 ```js
 const repl = require('repl');
 
-var replServer = repl.start({prompt: '> '});
+const replServer = repl.start({prompt: '> '});
 replServer.defineCommand('sayhello', {
   help: 'Say hello',
-  action: function(name) {
+  action: function sayHello(name) {
```

Comment:
I've just thought the docs should promote the style appropriate for the code base:

https://github.com/nodejs/node/pulls?q=is%3Apr+name+anonymous+functions+is%3Aclosed

But if I'm wrong, I'll roll back these two with just space insertion.